web-frontend/src/components/Review.vue

```diff
@@ -7,7 +7,7 @@
       <h3>Risk</h3>
       <ul>
         <li v-for="reason in dependency.risk_reasons" :key="reason">
-          {{ reason }}
+          <p v-html=reason></p>
```

Comment:
I think in Vuejs you can substitute `v-` with `:`

here that would be `:html="reason"`